scorekeeper/src/display/Display.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import Display from './Display';
+
+describe('The Display Component', () => {
+  
+  it('should render Display', () => {
+    render(<Display/>);
+    cleanup();
+  })
+
+  
```

Comment:
All of the extra white space around the test (`it`) block is unnecessary.  It's best to not have that kind of thing (and to be consistent with your white space), even if it doesn't make a difference!